vscode/README.md

```diff
@@ -29,6 +29,29 @@ See complete information about features [here](https://shopify.github.io/ruby-ls
 If you experience issues, please see the [troubleshooting
 guide](https://github.com/Shopify/ruby-lsp/blob/main/TROUBLESHOOTING.md).
 
+### Copilot chat agent
```

Comment:
"Copilot" is a heavily used term nowadays, let's clarify this is about "GitHub Copilot"

Reply:
Also, would it be worth explicitly saying this is an experimental feature? 

```suggestion
### [Experimental] GitHub Copilot chat agent
```